frontend/src/metabase/visualizations/visualizations/LinkViz/EntityDisplay.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+
+import Icon from "metabase/components/Icon";
+import { ItemIcon } from "metabase/search/components/SearchResult";
+import { color } from "metabase/lib/colors";
+import Ellipsified from "metabase/core/components/Ellipsified";
+import type { LinkEntity } from "./types";
+
+import { EntityDisplayContainer, LeftContainer } from "./EntityDisplay.styled";
+
+export const EntityDisplay = ({
+  entity,
+  showDescription = false,
+}: {
+  entity: LinkEntity;
+  showDescription?: boolean;
+}) => {
+  return (
+    <EntityDisplayContainer>
+      <LeftContainer>
+        <ItemIcon item={entity} type={entity?.model} active />
+        <Ellipsified>{entity?.name}</Ellipsified>
+      </LeftContainer>
+      {showDescription && entity?.description && (
+        <Icon
+          name="info"
+          color={color("text-light")}
+          tooltip={entity?.description}
+        />
+      )}
```

Comment:
The component shouldn't render if there's no description, so I believe we can remove the `?` when we pass the `tooltip` prop